applications/EigenSolversApplication/README.md

```diff
@@ -6,18 +6,18 @@ The *EigenSolversApplication* is a thin wrapper for the [Eigen linear algebra li
 
 The application provides the following direct sparse solvers:
 
-| Python class             | solver_type            | Matrix kind | Domain   | Dependencies |
-|--------------------------|------------------------|:-----------:|:--------:|:------------:|
-| SparseLUSolver           | `sparse_lu`            | Square      | Real     | None         |
-| SparseQRSolver           | `sparse_qr`            | Rectangular | Real     | None         |
-| SparseCGSolver           | `sparse_cg`            | SPD*        | Real     | None         |
-| PardisoLLTSolver         | `pardiso_llt`          | SPD*        | Real     | Intel® MKL   |
-| PardisoLDLTSolver        | `pardiso_ldlt`         | SPD*        | Real     | Intel® MKL   |
-| PardisoLUSolver          | `pardiso_lu`           | Square      | Real     | Intel® MKL   |
-| ComplexSparseLUSolver    | `sparse_lu_complex`    | Square      | Complex  | None         |
-| ComplexPardisoLLTSolver  | `pardiso_llt_complex`  | SPD*        | Complex  | Intel® MKL   |
-| ComplexPardisoLDLTSolver | `pardiso_ldlt_complex` | SPD*        | Complex  | Intel® MKL   |
-| ComplexPardisoLUSolver   | `pardiso_lu_complex`   | Square      | Complex  | Intel® MKL   |
+| Python class             | solver_type            | Matrix kind | Domain  | Dependencies |
```

Comment:
Please remove the changes. Centering is done by the Markdown syntax `:---:`.

Reply:
ok